test/README

```diff
@@ -9,3 +9,18 @@ symlinked from test/output/latest/html/index.html.
 
 *--output-dir*
 	Litani will write all of its test output files for this run to this path
+
+*--fast*
+    Do not run tests with SLOW variable set to True
+
+## Writing Tests
+
+Each test must set a variable called SLOW.
+We have 4 jobs per test (init, add-jobs, run-build, check-run) -
+If run time for all these 4 jobs for a test is <= 14s, then SLOW=False for that test else SLOW=True
+
+Added timeouts for the mentioned jobs is --fast is passed -
+    init - 2s
+    add-jobs - 2s
+    run-build - 8s
+    check-run - 2s
```

Comment:
> If run time for all these 4 jobs for a test is <= 14s, then SLOW=False for that test else SLOW=True

Not necessarily true with the breakdown of the timeouts. If add-jobs takes 3 seconds but run-build only takes 5, the runtime is <=14 but the jobs would still time out. 

```
Each test is made up of 4 jobs: init, add-jobs, run-build, and check-run
If SLOW is set, the jobs are given the following timeouts
    init - 2 seconds
    add-jobs - 2 seconds
    run-build - 8 seconds
    check-run - 2 seconds
```